README.md

```diff
@@ -17,8 +17,10 @@ For job messages to appear on Treeherder production, the destination of `tc-tree
 be used.  Also, `tc-treeherder-staging` could be used for reporting jobs to the Treeherder
 staging environment.
 
-Note: Github repos should use the `project` form of `<user>/<project>` to be recognized
-as a github source.
+Github repos should use the `project` form of `<owner>/<repository>` to be recognized
+as a github source. The `owner` field is unused. The `repository` field is the repository name
```

Comment:
Unused by treeherder itself, you mean?

Reply:
Yes,  see also https://github.com/mozilla/treeherder/pull/4327. Is there anything else that reads the Pulse messages generated taskcluster-treeherder?

Reply:
No, that's about it.  My next question was going to be "is that information also in treeherder" but you've pre-empted me :)